src/ContainerResolver.php

```diff
@@ -99,4 +129,43 @@ private static function resolveMvcContainer(): ContainerInterface
 
         return $serviceManager;
     }
+
+    /**
+     * @psalm-param non-empty-string $containerPath
+     */
+    private function resolveContainerFromAbsolutePath(string $containerPath): ContainerInterface
+    {
+        if (! file_exists($containerPath)) {
+            throw new InvalidArgumentException(
+                sprintf(
+                    'Provided container path could not be resolved to an existing file: %s',
+                    $containerPath
+                )
+            );
+        }
+
+        $container = include $containerPath;
+        Assert::isInstanceOf($container, ContainerInterface::class, 'Failed to load PSR-11 container');
+
+        return $container;
+    }
+
+    /**
+     * Verifies that the provided path does not contain an absolute path.
+     * Absolute paths can be either start with `/` or provided as an URI.
+     *
+     * @psalm-param non-empty-string $pathToContainer
+     */
+    private function isAbsolutePath(string $pathToContainer): bool
+    {
+        if ($pathToContainer[0] === '/') {
+            return true;
+        }
+
+        if (strpos($pathToContainer, '://') !== false) {
```

Comment:
is it working with remote container configuration?

Reply:
Thats mainly for the unit tests where virtual streams are used, but yes, if the stream is supported by PHP, this would work. But actually I would not expose that as an "official" feature. Imho passing locally stored files should be preferred.